src/store/actions.js

```diff
@@ -0,0 +1,182 @@
+import Vue from 'vue';
+import axios from 'axios'
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+let $xmltojson = require('../xml2json.min.js')
+let GOODREADS_API_KEY = process.env.GOODREADSKEY // Add your api key here
+
+
+console.log(process.env.GOODREADSKEY || "Fallback");
```

Comment:
If you have verified that the code works, then you could now remove the console.log() line.